jme3-examples/src/main/java/jme3test/texture/TestTextureArrayCompressed.java

```diff
@@ -31,8 +31,8 @@ public void simpleInitApp()
        }
        
        
-       Texture tex1 = assetManager.loadTexture( "Textures/Terrain/Pond/Pond.dds");
-       Texture tex2 = assetManager.loadTexture("Textures/Terrain/BrickWall/BrickWall.dds");
+       Texture tex1 = assetManager.loadTexture( "Textures/Terrain/Pond/Pond_dxt5.dds");
+       Texture tex2 = assetManager.loadTexture("Textures/Terrain/BrickWall/BrickWall_dxt5.dds");
```

Comment:
Why?

Reply:
I created non-compressed dds files for most existing dds and renamed the previous to file_format.dds so if running any regular test non-compressed image is used being able to run in desktop and mobiles while this test which is related to compressed images I needed to change it to keep using the compressed image

Reply:
Ok, can we keep only the non compressed version for every texture except the ones used in tests that require compressed textures?

Reply:
Yes, compressed dds's not being used in tests could be removed. These two textures are the only compressed textures used